src/main/java/frc/robot/subsystems/drivetrain/commands/AdjustToTargetOnCommand.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.Robot;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+public class AdjustToTargetOnCommand extends CommandBase {
```

Comment:
Can be renamed to `AdjustToTarget` (it's in commands package so its obvious that it's a command)

Reply:
It's not AdjustToTargetCommand it's AdjustToTargetOnCommand.